projects/questionset-editor-library/src/lib/components/match/match.component.ts

```diff
@@ -0,0 +1,165 @@
+import { Component, Input, OnInit, OnChanges, Output, EventEmitter, SimpleChanges } from '@angular/core';
+import * as _ from 'lodash-es';
+import { EditorTelemetryService } from '../../services/telemetry/telemetry.service';
+import { ConfigService } from '../../services/config/config.service';
+
+@Component({
+  selector: 'lib-match',
+  templateUrl: './match.component.html',
+  styleUrls: ['./match.component.scss']
+})
+export class MatchComponent implements OnInit, OnChanges {
+  @Input() editorState: any;
+  @Input() showFormError;
+  @Input() sourcingSettings;
```

Comment:
@gupta-arpan `sourcingSettings` is not getting used anywhere. Please remove it from this component.

Reply:
completed